packages/client/components/Dashboard/MobileDashSidebar.tsx

```diff
@@ -134,6 +134,12 @@ const MobileDashSidebar = (props: Props) => {
                 href={`/me/organizations/${orgId}/${MEMBERS_PAGE}`}
                 label={'Members'}
               />
+              <LeftDashNavItem
```

Comment:
The [designs](https://www.figma.com/file/SyJdkzumWsUJo58cmBGgus/Checkout-concepts?type=design&node-id=161-2088&t=xvkm9h4kV0Or6GQi-0) have Organization Details at the top of the sidebar, but I think the billing page should be the top priority, and this should be the lowest priority, so I've put it at the bottom of the sidebar.

Let me know if you disagree @acressall 🙏

Reply:
Oooh I like this idea. Maybe later we can change the name to "organization settings" since it will include the AI toggle and potentially other things

Reply:
Great! Updated to Organization Settings 👍